ee/tabby-ui/components/textarea-search.tsx

```diff
@@ -199,57 +250,79 @@ export default function TextAreaSearch({
       </div>
       <div
```

Comment:
If the value of modelInfo.chat is null, consider not displaying the bottom toolbar.